.changeset/two-oranges-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vue': minor
```

Comment:
```suggestion
'@astrojs/vue': patch
```

imo this is a bug fix. It'd be a feature if it was adding runtime support for this, but in my mind differences between runtime and editor are bugs